src/components/ResourceList/components/Item/Item.scss

```diff
@@ -125,6 +125,14 @@
   margin: (-1 * resource-list-item(padding)) spacing(extra-tight)
     (-1 * resource-list-item(padding)) resource-list-item(control-indent);
   display: flex;
+
+  @include breakpoint-before(resource-list(breakpoint-small)) {
+    visibility: hidden;
```

Comment:
`visibility: hidden;` is required here to reserve the space in the dom so the item doesn't collapse and become partially hidden

Reply:
I wonder what changed that this wasn't an issue before.